integration/show_open_test.go

```diff
@@ -0,0 +1,3 @@
+package integration
+
+// TODO: what is a good integration test for "show-open"?
```

Comment:
@naggie Can you recommend a strategy for these tests?

Reply:
I suppose just checking that the expected number of tasks are presented for the given context. I suppose with JSON output, as there's no terminal height limitation the `next` and `show-open` are effectively the same.